### ISSUE TITLE
Minor error when triggering person states

### ISSUE THREAD

**Issue by systemcrash** [basnijholt/adaptive-lighting#170]
#### Version information:
1.0.14

#### Description:

In developer tools, when testing triggers of home, not_home state for a few persons, triggering home after not_home (this integrates with OpenWRT, but just setting the state in dev tools triggered the error), I got the below error. Cannot consistently reproduce. Last few tries seem to be OK. 

```
ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/adaptive_lighting/switch.py", line 944, in _light_event
    if await self.turn_on_off_listener.maybe_cancel_adjusting(
  File "/config/custom_components/adaptive_lighting/switch.py", line 1445, in maybe_cancel_adjusting
    id_turn_on = turn_on_event.context.id
AttributeError: 'NoneType' object has no attribute 'context'
```

Edit: These states normally trigger automation scripts. But I was just testing via dev tools.

**Comment by systemcrash** [comment]
Got this again with a simple on/off of my lights:

ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/adaptive_lighting/switch.py", line 944, in _light_event
    if await self.turn_on_off_listener.maybe_cancel_adjusting(
  File "/config/custom_components/adaptive_lighting/switch.py", line 1445, in maybe_cancel_adjusting
    id_turn_on = turn_on_event.context.id
AttributeError: 'NoneType' object has no attribute 'context'
2021-08-16 23:12:43 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/adaptive_lighting/switch.py", line 944, in _light_event
    if await self.turn_on_off_listener.maybe_cancel_adjusting(
  File "/config/custom_components/adaptive_lighting/switch.py", line 1445, in maybe_cancel_adjusting
    id_turn_on = turn_on_event.context.id
AttributeError: 'NoneType' object has no attribute 'context'

MY on/off is triggered  via an IKEA TRÅDFRI controller.. if that makes a difference.

**Comment by RubenKelevra** [comment]
Not sure I understand what you're trying to do.

Can you show us which automation / service calls you're using and how your adaptive_lighting is configured?

**Comment by basnijholt** [comment]
This issue should be fixed by #325.

Please update to 1.19.0 :)
